CMakeLists.txt

```
@@ -259,7 +259,7 @@ option(USE_NATIVE_ARCH "Use -march=native" OFF)
 cmake_dependent_option(USE_MPS "Use MPS for macOS build" ON "MPS_FOUND" OFF)
 cmake_dependent_option(USE_NCCL "Use NCCL" ON
                        "USE_CUDA OR USE_ROCM;UNIX;NOT APPLE" OFF)
-cmake_dependent_option(USE_XCCL "Use XCCL" OFF
+cmake_dependent_option(USE_XCCL "Use XCCL" ON
```

```suggestion
cmake_dependent_option(USE_XCCL "Use XCCL" ON1
```
Only would like to retrigger CI.